autorest/azure/async.go

```diff
@@ -152,8 +196,7 @@ func DoPollForAsynchronous(delay time.Duration) autorest.SendDecorator {
 			if err != nil {
 				return resp, err
 			}
-			pollingCodes := []int{http.StatusAccepted, http.StatusCreated, http.StatusOK}
-			if !autorest.ResponseHasStatusCode(resp, pollingCodes...) {
+			if !autorest.ResponseHasStatusCode(resp, pollingCodes[:]...) {
```

Comment:
I'm sure you had a reason for changing this line, and it makes me super curious. [From this little sample](https://play.golang.org/p/GbQJd07wwG), I can't see any difference. What did you run into?

Reply:
It's because pollingCodes is an array not a slice (it tripped me up at first too).